lib/web_console/whiny_object.rb

```diff
@@ -0,0 +1,23 @@
+module WebConsole
+  class WhinyObject < SimpleDelegator
```

Comment:
I have been thinking about this for a while: it may be better if we drop the decorators and use `ActiveSupport::Notifications.instrument` for logging. This can happen in the middleware and we can create a subscriber that logs the information if the users set `whiny_requests` to true.

This will drop the decorators as a whole. It didn't matter that much when we had just one object that we logged, but now that we have more I don't think the extra overhead is worth it. @jmbejar WDYT?